src/test/resources/debug-etc/config.yml

```diff
@@ -27,3 +27,22 @@ logging:
     - type: file
       archive: false
       currentLogFilename: data/dd-verify-file-migration.log
+
+easyDb:
+  driverClass: org.postgresql.Driver
+    url: jdbc:postgresql://deasy.dans.knaw.nl:5432/easy_db
+    logValidationErrors: true
+    properties:
+      hibernate.dialect: 'org.hibernate.dialect.PostgreSQL92Dialect'
+      hibernate.hbm2ddl.auto: update
```

Comment:
This database is used just read only. Do we need this line?

Reply:
I suppose we also need to configure a use and password.

https://github.com/DANS-KNAW/easy-app/blob/f3871a5dcdcdbb9d043558238f85e10eb3c41c9e/front-end/easy-webui/src/main/assembly/dist/cfg/application.properties#L53-L57